### ISSUE TITLE
\varnothing instead of \emptyset

### ISSUE THREAD

**Issue by kulla** [vroland/mwlint#32]
In Formeln: `\varnothing` instead of `\emptyset`.

**Comment by vroland** [comment]
Why is `\emptyset` bad? Sounds more intuitive to me...

**Comment by Lodifice** [comment]
I think instead of choosing the wrong icon we could instead change the math font. There are also some typographical errors in our formulas. I suggest using the package "unicode-math", which is IMHO the way to go with LuaLaTeX.

Unfortunately, it only supports a few fonts, see https://ctan.org/pkg/unicode-math.
For example, TeX Gyre Pagella Math has a nicer glyph for the empty set than the standard math font which we use at the moment.

**Comment by kulla** [comment]
@Lodifice Good point. I have created an issue for this ( https://github.com/vroland/mfnf-export/issues/9 / this issue is for the linter ).

**Comment by vroland** [comment]
@kulla Unfortunately, we should not treat PDF-export and HTML as separate issues any more. As HTML will become an important export target, we have to consider wether it is possible to render these correctly in a browser, where it is harder to use custom math fonts (afaik). See also https://katex.org/.
Still, I would prefer `\emptyset` for semantics...

**Comment by vroland** [comment]
Hm, this looks like another case where a good formula parser would be nice. We could automatically replace this for the export....
